### PR TITLE
Removed mainapplication layer

### PR DESCRIPTION
**Pull request by calixtus**
Should clean up appdirs etc.:
grafik

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

**Comment by Siedlerchr** [comment]
Needs to be ajusted in the gradle file as well:
> Fehler: Hauptklasse org.jabref.cli.Launcher konnte in Modul org.jabref nicht gefunden oder geladen werden

when I try to run the jpackaged binary

File: src/main/java/org/jabref/Launcher.java [hunk 1]
```diff
@@ -148,7 +149,7 @@ private static void addLogToDisk() {
     }
 
     private static void initializeLogger() {
-        LOGGER = LoggerFactory.getLogger(MainApplication.class);
+        LOGGER = LoggerFactory.getLogger(JabRefGUI.class);
```
Comment (koppor) [inline src/main/java/org/jabref/Launcher.java:152]: Oh, Logger for `Launcher` called `JabRefGUI`. Really?

File: src/main/java/org/jabref/Launcher.java [hunk 2]
```diff
@@ -167,7 +168,7 @@ private static boolean handleMultipleAppInstances(String[] args, RemotePreferenc
                 if (remoteClient.sendCommandLineArguments(args)) {
                     // So we assume it's all taken care of, and quit.
                     LOGGER.debug("Arguments passed on to running JabRef instance.");
-                    System.out.println(Localization.lang("Arguments passed on to running JabRef instance. Shutting down."));
+                    LOGGER.info(Localization.lang("Arguments passed on to running JabRef instance. Shutting down."));
```
Comment (koppor) [inline src/main/java/org/jabref/Launcher.java:171]: Please revert - and add a comment that "System.out.println" is used for user output. (Line above is the logger - this line was really intended for a script-use output etc)

File: src/main/java/org/jabref/gui/MainApplication.java [hunk 3]
```diff
@@ -1,54 +0,0 @@
...
-        new JabRefGUI(mainStage, uiCommands, preferences, fileUpdateMonitor);
-    }
-
-    @Override
-    public void stop() {
-        OOBibBaseConnect.closeOfficeConnection();
-        Globals.stopBackgroundTasks();
-        Globals.shutdownThreadPools();
-        try {
-            Globals.predatoryJournalRepository.close();
```
Comment (koppor) [inline src/main/java/org/jabref/gui/MainApplication.java:49]: Where did this go? I could not find it in `JabRefGUI`.

**Comment by Siedlerchr** [comment]
binary works now

File: src/main/java/org/jabref/gui/MainApplication.java (hunk 3 above)
Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/MainApplication.java:49]: LIne 100 in JabrefGUI

File: src/main/java/org/jabref/Launcher.java (hunk 1 above)
Comment (calixtus) [reply src/main/java/org/jabref/Launcher.java:152]: Yes, seemed suspicious to mee to, but I just adapted it to the new architecture. Will change to `Launcher.class`.

**Review by koppor — APPROVED** [review]

**Review by Siedlerchr — APPROVED** [review]

**Pull request merged**